src/OptimisticTokenVotingPlugin.sol

```diff
@@ -542,6 +557,7 @@ contract OptimisticTokenVotingPlugin is
 
         // No more L2 vetoes can be registered
         // return false if no L2 votes have been aggregated until now
+        // Q(danielw): what if the aggregated L2 votes is actually zero? Maybe use another boolean var to track?
```

Comment:
Not sure I follow
You mean using a new proposal variable, which is already precomputed following a similar logic while removing this function?

Reply:
can `proposal_.aggregatedL2Balance == 0` mean EITHER the actual L2 veto is zero OR somehow the actual L2 veto was not bridged/aggregated to L1 for this proposal (for example, L2 is down)? If this is the case, shall we differentiate these two cases?

Reply:
Oh I see. This edge case was briefly commented on the chat without a clear outcome. Since this doesn't affect the future-proofness of the plugin it wasn't brought back again. 

Intuitively, if there are 0 aggregated L2 votes and the L2 is currently down, we should only use the L1 census. But I wonder if this could lead to unforeseen race conditions (someone knowing that there is an upcoming L2 batch that somehow can't make it to the L1, and getting to execute with a favorable L1 census)

Reply:
I think we can address this later before we enable L2 voting. May I trouble you to add a comment in the code?

Reply:
@dantaik Sure, let's add a comment for this

I am currently addressing the review's feedback on this separate branch https://github.com/aragon/taiko-contracts/tree/f/review-feedback as your changes are living on a separate repo